### DISCUSSION TITLE
wiki/help-pages/mod-platform/

### DISCUSSION THREAD

**Discussion by giscus** [PrismLauncher/prismlauncher.org#645] [Wiki]
# wiki/help-pages/mod-platform/

An Open Source Minecraft launcher with the ability to manage multiple instances, accounts and mods. Focused on user freedom and free redistributability.

https://prismlauncher.org/wiki/help-pages/mod-platform/

**Comment by PwRhGaStPY** [comment]
I'm trying to download Lucky Block for 1.20.1 from CurseForge, but the version selector lists "No valid version found."
I would guess this is when downloading via browser is required, but is there any way to get around that? 

I previously posted this comment on [a different page](https://prismlauncher.org/wiki/help-pages/flame-platform/) before realizing it was the wrong one.

**Reply by PwRhGaStPY** [reply]
I have 1.20.1 selected as my Minecraft version, I made sure of that.

**Reply by ghost** [reply]
I think there is no workaround.

**Comment by Serega007RU** [comment]
it doesn't work, it says that "No valid version found" although this version definitely exists on CurseForge. I'm trying to download Hardcore True Darkness for Forge 1.20.1

**Comment by Cashi-Best** [comment]
can we please be told where it is?

**Comment by VOIDVLATS** [comment]
i can't find ZombieMania when i search it, can someone help me?

**Comment by snowdevrbx-cyber** [comment]
when i press "install mods" its 2 errors and i can fix im reinstall and nothing
